src/actions/settingsActions.ts

```diff
@@ -18,11 +18,11 @@ const persistentStore = new Store({ name: 'pc-nrfconnect-programmer' });
 export function loadSettings() {
     return (dispatch: TDispatch) => {
         const settings = persistentStore.get('settings') || {};
-        if (!settings.autoRead) {
+        if (settings.autoRead == null) {
             settings.autoRead = false;
```

Comment:
Use nullish assignment?

Reply:
So for example `settings.autoRead = settings.autoRead ?? false`?